FND/Accounting/ISO4217-CurrencyCodes.rdf

```diff
@@ -34,12 +34,15 @@
 	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/FND/Accounting/ISO4217-CurrencyCodes/">
 		<rdfs:label>ISO 4217-1 Currency Codes Ontology</rdfs:label>
 		<dct:abstract>This ontology represents the subset of the ISO 4217 standard that include the actual currency codes.</dct:abstract>
+		<dct:issued rdf:datatype="&xsd;dateTime">2021-10-01T00:00:00&gt;</dct:issued>
```

Comment:
```suggestion
		<dct:issued rdf:datatype="&xsd;dateTime">2021-10-01T00:00:00</dct:issued>
```

This causes an error in the _Hygiene Tests_ - see [consoleText for PR-1683 build 3](https://jenkins.edmcouncil.org/job/fibo/view/change-requests/job/PR-1683/3/consoleText):
`Lexical form '2021-10-01T00:00:00>' not valid for datatype XSD dateTime`

Reply:
> This causes an error in the _Hygiene Tests_ - see [consoleText for PR-1683 build 3](https://jenkins.edmcouncil.org/job/fibo/view/change-requests/job/PR-1683/3/consoleText): `Lexical form '2021-10-01T00:00:00>' not valid for datatype XSD dateTime`

Technically, it is a warning (from riot app), not an error, but definitely needs to be fixed.

Reply:
Yes, fixed